pkg/assetattributor/cloudassetinventory.go

```diff
@@ -92,8 +92,6 @@ func (n *CloudAssetInventory) Attribute(ctx context.Context, asset domain.Nexpos
 	if asset.IP == "" && asset.Hostname == "" {
 		return domain.NexposeAttributedAssetVulnerabilities{}, domain.AssetNotFoundError{
 			Inner:          fmt.Errorf("asset has no IP or hostname"),
-			AssetID:        fmt.Sprintf("%d", asset.ID),
```

Comment:
Please use `strconv.Itoa` function instead of `fmt.Sprintf`. https://golang.org/pkg/strconv/#Itoa

Reply:
this line is deleted :/